src/events/SocialMedia.ts

```diff
@@ -9,9 +10,10 @@ export default defineEvent({
   `,
 
   apply(context) {
-    const g = context.state.game
-    if (g) {
-      g.values.popularity += Math.max(-g.values.popularity, -20 as Percent)
+    const game = { ...context.state.game } as Game
```

Comment:
Could we change the signature to something more practical like in Law or provide helpers?

Reply:
Yes, I find this also very unpleasant to always have to copy the whole `Game` and deep access values.
I have created #84 to make that easier.